core/src/main/scala/com/databricks/labs/remorph/transpilers/Transpiler.scala

```diff
@@ -4,11 +4,12 @@ import com.databricks.labs.remorph.parsers.{intermediate => ir}
 
 trait Transpiler {
   def transpile(input: String): String
+  def parse(input: String): ir.LogicalPlan
```

Comment:
i am not sure we have to add `parse` method to this public interface.

Reply:
I need `parse` output to create DAG? should I create a `parse` object and refactor all transpiler with that?